hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/auth/HugeAuthenticator.java

```diff
@@ -105,7 +105,6 @@ default User authenticate(final Map<String, String> credentials)
         HugeGraphAuthProxy.logUser(user, credentials.get(KEY_PATH));
         /*
          * Set authentication context
-         * TODO: unset context after finishing a request
          */
```

Comment:
change multi to one line comment -like-> `Set authentication context & unset in AccessLogFilter`?